.github/print-integration-test-selection

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python3
```

Comment:
file name `print-integration-test-selection` -> `print-integration-test-selection.py`?
or it has to be `print-integration-test-selection`

Reply:
Some use extensions for executable scripts, some don't and rely on a) executable bit and b) shebang line. For instance, `pip` and `pipenv` subscribe to the latter religion:
```
% ls -l /usr/bin/pip3
-rwxr-xr-x 1 root root 366 Nov  3 13:08 /usr/bin/pip3
% head -1 /usr/bin/pip3
#!/usr/bin/python
```